app/assets/javascripts/initializers/initScrolling.js

```diff
@@ -44,6 +44,9 @@ function fetchNext(el, endpoint, insertCallback) {
 
 function insertNext(params, buildCallback) {
```

Comment:
Hmm, just done a brief look at the code only and I think you may be right.

I'll take a more detailed look later, but I suspect `matchingAction` will be `false` and therefore prevent the DOM being updated, unless `containerAction` happens to `undefined`.

I'll fix it if that's the case, otherwise I'll try and make the code more consistent.

Reply:
Thank you ✨

Reply:
Sorry, been a bit busy.

I checked up on this and fortunately there was no bug because both `containerAction` and `params.action` were `undefined`, so `matchingAction` was `true` 😌 

I didn't want to make too many changes (i.e. changing the HTML to add a new action param) but hopefully my latest commit (80ae4b2) makes it clearer that this case is being handled

Reply:
No worries and Thanks so much @tomalexhughes!